portal/templates/index.njk

```diff
@@ -16,69 +16,81 @@
     <link rel="apple-touch-icon" sizes="152x152" href="/assets/images/govuk-apple-touch-icon-152x152.png">
     <link rel="apple-touch-icon" href="/assets/images/govuk-apple-touch-icon.png">
     <link rel="stylesheet" href="/assets/css/styles.css"/>
-    <meta property="og:image" content="/assets/images/govuk-opengraph-image.png">
-    {% block head %}{% endblock %}
```

Comment:
We need this block head for post-login-redirect.njk to operate correctly

Reply:
Not sure what happened here, reverted